helm/app-updater-app/templates/cronjob.yaml

```diff
@@ -15,9 +15,9 @@ spec:
           serviceAccount: app-updater
           restartPolicy: Never
           containers:
-          - image: teemow/debug
+          - image: quay.io/giantswarm/debug:update
```

Comment:
hmm what about use semver in the tagging?

Reply:
Will change to a specific image, as Timo suggested